static/main.js

```diff
@@ -1555,12 +1549,12 @@ class App {
             this.gamePages.$edit.push();
         })
         this.socket.on("sNextTurn", data => {
-            this.game.state = "wait";
+            this.game.stage = "wait";
```

Comment:
Кажется, здесь должно быть так:
```suggestion
            this.game.stage = "play_wait";
```